proto/src/determined/experiment/v1/searcher.proto

```diff
@@ -35,9 +35,6 @@ enum RunnableType {
   RUNNABLE_TYPE_TRAIN = 1;
   // Signals to a trial runner it should compute validation metrics.
   RUNNABLE_TYPE_VALIDATE = 2;
-  // Signals to the trial runner that the current model state should be
-  // checkpointed.
-  RUNNABLE_TYPE_CHECKPOINT = 3;
```

Comment:
q: will this be breaking?

Reply:
yeah, but only breaking for `det preview-search` (which reminds me there is CLI code I can remove too)

Reply:
breaking for `det preview-search` consumed via the new rest api (which I think we're _probably_ safe on)